src/views/results/index.ts

```diff
@@ -254,9 +249,29 @@ export function initialise(context: vscode.ExtensionContext) {
   )
 }
 
+export function statementMiddleware(statement: StatementInfo) {
+  switch (statement.type) {
+    case StatementType.Create:
+    case StatementType.Alter:
+      const ref = statement.refs[0];
+      const databaseObj =
+        statement.type === StatementType.Create && ref.createType.toUpperCase() === `schema`
```

Comment:
This will never be true, right?
`ref.createType.toUpperCase() === 'schema'`

Reply:
Good spot. I will fix this